### PR TITLE
First build

### PR DESCRIPTION
**Pull request by akramkazmi71**
First build.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,26 +1,76 @@
...
+        this.setState({
+          list: tempArr,
+          size: response.total_count
+        })
+      });
+  }
+    
+  nextPage = () => {
+    var x=this.state.size/30
+    this.state.max_page=Math.floor(x)+1
```
Comment (arshadkazmi42) [inline src/App.js]: Create a local variable for max_page as it is not used any where else.

Also, use `maxPage` in javascript camelCase is followed

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,76 @@
...
+    this.state.pageNumber=this.state.pageNumber+1
```
Comment (arshadkazmi42) [inline src/App.js]: Do not modify state like this.
It should be always updated using setState
```
this.setState({
  pageNumber: this.state.pageNumber + 1
});
```

https://stackoverflow.com/a/37760774/3136282

File: src/App.js [hunk 3]
```diff
@@ -1,26 +1,76 @@
...
+    if(this.state.pageNumber<=this.state.max_page){
+      this.callPage();
+    }
+  }
+
+  previousPage = () => {
+    this.state.pageNumber-=1
```
Comment (arshadkazmi42) [inline src/App.js]: Same as above, do no update state directly.
Use setState

File: src/App.js [hunk 4]
```diff
@@ -1,26 +1,76 @@
...
+    this.state.count-=(this.state.count%30)+30
```
Comment (arshadkazmi42) [inline src/App.js]: Same as above

File: src/components/List.js [hunk 5]
```diff
@@ -0,0 +1,29 @@
+import React, {Component} from 'react';
```
Comment (arshadkazmi42) [inline src/components/List.js]: This can be deleted, as this is not getting used

Comment (akramkazmi71) [reply src/components/List.js]: I am extending the App class with Component.By removing this I would get an error.

Comment (arshadkazmi42) [reply src/components/List.js]: I mean delete the `List.js` file.

**Pull request merged**